release/publish.sh

```diff
@@ -40,7 +40,7 @@ VERSION="$(cat "${WD}/trigger-publish")"
 SOURCE_GCS_BUCKET=${SOURCE_GCS_BUCKET:-istio-prerelease/prerelease}
 GCS_BUCKET=${GCS_BUCKET:-istio-release/releases}
 HELM_BUCKET=${HELM_BUCKET:-istio-release/charts}
-HELM_BUCKET_RELEASE=${HELM_BUCKET:-gcr.io/istio-release/charts}
+HELM_HUB_RELEASE=${HELM_HUB_RELEASE:-gcr.io/istio-release/charts}
```

Comment:
Does line 71 also need a change with this?

Reply:
yes ty

Reply:
Thanks John. Seems like this should be good (adding the gcr.io to what was there).